css/base/variables.css

```diff
@@ -257,7 +257,7 @@ body {
 
   /* File Upload Items */
   --file-upload-border: var(--border);
-  --file-upload-border-left-size: var(--border-width-large);
+  --file-upload-border-inline-start-size: var(--border-width-large);
```

Comment:
I'm not certain we want to change the variable names. Though I understand the reason for it... It would be good to discuss.

Reply:
I think I'll go with this approach:

> Solutions - 2: Mapped variables

> We could create new variables and map those to the old ones, such as:

>    `--quote-padding-inline-start: var(--quote-padding-left);`

Reply:
@maria I think this is fixed now.

We now have
1. The old variable so we don't break existing sites, and
2. The new variable for how we will work from here on ...